py-polars/polars/internals/expr/list.py

```diff
@@ -234,10 +234,7 @@ def concat(
             return self.concat(pli.Series([other]))
 
         other_list: list[pli.Expr | str | pli.Series]
-        if not isinstance(other, list):
-            other_list = [other]
-        else:
-            other_list = copy.copy(other)  # type: ignore[arg-type]
+        other_list = [other] if not isinstance(other, list) else copy.copy(other)  # type: ignore[arg-type]
```

Comment:
Lines like this are the main reason I prefer `W505` over `E501`. We can add the mypy ignore comment without having to worry about line length.